JS.md

```diff
@@ -361,6 +365,7 @@ each [external](https://github.com/WebAssembly/spec/blob/master/interpreter/spec
 1. If `e` is a [global](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/instance.ml#L15) `v`:
   1. [Assert](https://tc39.github.io/ecma262/#assert): the global is immutable
      by MVP validation constraint.
+  1. If `v` is an `i64`, throw a `WebAssembly.LinkError`.
```

Comment:
Since we can't instantiate one wasm module knowing that its exports will only be used by another wasm module, unfortunately I don't think there's anything we can do in the MVP.  If we either add "`WebAssembly.instantiateGraph`" (which takes and simultaneously instantiates multiple modules, allowing them to reference each other) or we add integration with ES6 modules, we could allow this to work b/c we could know that "both sides" were wasm.

Practically speaking, I think this is fine b/c the only need for global imports in the MVP is to provide (32-bit) offsets/pointers for dynamic linking purposes; actual C++ `int64_t` global variables will go into linear memory.